app/views/tv_series_viewings/_buttons_toggle.html.erb

```diff
@@ -0,0 +1,17 @@
+<div id='js_tv_series_viewings_buttons_toggle'>
+  <% if current_user.tv_series_viewings.active.find_by(show_id: show_id).present? %>
+    <div class='manage-screenings'>
+      <%= button_to '<span class="material-symbols-outlined">bookmark_remove</span> Mark as Finished Watching'.html_safe, 
+        tv_series_viewing_path(id: show_id, show_id: show_id, title: title),
```

Comment:
This `id: show_id` is... icky. Since this is a `:patch` request, rails is expecting the id of the `tv_series_viewing` record. Well, we don't have one here. We don't use that id in the controller either. So I'm fudging it here. However, this does feel like it is wafting up a smell. I'm not sure how to handle it yet. @mikevallano any ideas?

Reply:
🤔 it looks like we _could_ have it since we're doing `if current_user.tv_series_viewings.active.find_by(show_id: show_id).present?` on line 2?  but dunno how that would work with turbo, etc.

regardless, i think the way you have it is totally fine.